draft-ietf-httpbis-message-signatures.md

```diff
@@ -1013,16 +1018,32 @@ Use of this algorithm can be indicated at runtime using the `hmac-sha256` value
 To sign using this algorithm, the signer applies the `ECDSA` algorithm {{FIPS186-4}} using curve P-256 with the signer's private signing key and
 the signature input string ({{create-sig-input}}).
 The hash SHA-256 {{RFC6234}} is applied to the signature input string to create
-the digest content to which the digital signature is applied.
-The resulting signed content byte array is the HTTP message signature output used in {{sign}}.
+the digest content to which the digital signature is applied, (`M`).
+The signature algorithm returns two integer values, `r` and `s`. These are both encoded in big-endian unsigned integers, zero-padded to 32-octets each. These encoded values are concatenated into a single 64-octet array consisting of the encoded value of `r` followed by the encoded value of `s`. The resulting concatenation of `(r, s)` is byte array of the HTTP message signature output used in {{sign}}.
```

Comment:
You might want to add an informative ref to JWA, https://www.rfc-editor.org/rfc/rfc7518#section-3.4